File: app/src/main/java/com/example/tipper/WelcomeScreenActivity.java
package com.example.tipper;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import android.widget.Button;

//Autor Karol Kuchnio s21912
public class WelcomeScreenActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcome_screen);

        Button button1 = findViewById(R.id.button1);
        Button button2 = findViewById(R.id.button2);
        Button button3 = findViewById(R.id.button3);
        Button quizButton = findViewById(R.id.button4);

        button1.setOnClickListener(view -> {
            Intent intent = new Intent(WelcomeScreenActivity.this, BMIActivity.class);
            startActivity(intent);
        });

        button2.setOnClickListener(view -> {
            Intent intent = new Intent(WelcomeScreenActivity.this, CaloriesCalcActivity.class);
            startActivity(intent);
        });
        button3.setOnClickListener(view -> {

            Intent intent = new Intent(WelcomeScreenActivity.this, ActivityRecipe.class);
            startActivity(intent);
        });

        quizButton.setOnClickListener(view -> {

            Intent intent = new Intent(WelcomeScreenActivity.this, QuizActivity.class);
            startActivity(intent);
        });


    }

}

File: app/src/main/java/com/example/tipper/WelcomeScreenActivity.kt
package com.example.tipper

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
//Autor Karol Kuchnio s21912
class WelcomeScreenActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcome_screen)
        val button1 = findViewById<Button>(R.id.button1)
        val button2 = findViewById<Button>(R.id.button2)
        val button3 = findViewById<Button>(R.id.button3)
        val quizButton = findViewById<Button>(R.id.button4)
        button1.setOnClickListener { view: View? ->
            val intent = Intent(this@WelcomeScreenActivity, BMIActivity::class.java)
            startActivity(intent)
        }
        button2.setOnClickListener { view: View? ->
            val intent = Intent(this@WelcomeScreenActivity, CaloriesCalcActivity::class.java)
            startActivity(intent)
        }
        button3.setOnClickListener { view: View? ->
            val intent = Intent(this@WelcomeScreenActivity, ActivityRecipe::class.java)
            startActivity(intent)
        }
        quizButton.setOnClickListener { view: View? ->
            val intent = Intent(this@WelcomeScreenActivity, QuizActivity::class.java)
            startActivity(intent)
        }
    }
}
